ts/src/base/Exchange.ts

```diff
@@ -1261,11 +1261,13 @@ export default class Exchange {
 
     async close () {
         const clients = Object.values (this.clients || {});
+        const closedClients = [];
         for (let i = 0; i < clients.length; i++) {
             const client = clients[i] as WsClient;
             delete this.clients[client.url];
-            await client.close ();
+            closedClients.push(client.close ());
```

Comment:
@pcriadoperez hum is this an optimization? because the previous code was closing them sequentially and now we do it "in parallel" but not sure how it relates to the issue

Reply:
Correct, its an optimization and does not relate to this issue